cmake/double-conversionConfig.cmake

```diff
@@ -0,0 +1 @@
+include("${CMAKE_CURRENT_LIST_DIR}/double-conversionTargets.cmake")
```

Comment:
Please add `@PACKAGE_INIT@` and `check_required_components`. See example:
* https://github.com/forexample/package-example/blob/master/Foo/cmake/Config.cmake.in

Reply:
Sure

Reply:
I guess we don't need this file anymore.